HHS/Head-Start-TTADP
# [TTAHUB-605] Add filter count to filters button

## Description of change

Add filter count to filters button.

## How to test

Go to any page that uses the new filters. If a filter is selected by default this should show in the filter button text as "Filters (x)". Adding and removing filters should also update this number.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-605


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy


### After merge/deploy
